domains/xrap1dx.json

```
@@ -0,0 +1,12 @@
+{
+    "description": "making a cool portfolio website.",
+    "repo": "https://github.com/xrap1dx?tab=repositories",
+    "owner": {
+        "username": "xrap1dx",
+        "email": "sirrapid155@gmail.com"
+    },
+    "record": {
+        "TXT": "Hello World!",
+        "A": ["69.30.249.53"]
```

While we are here, is it possible if you just remove TXT line since its not being used for verification stuff yet?
```suggestion
        "A": ["69.30.249.53"]
```